src/i_pcsound.c

```diff
@@ -38,7 +38,7 @@ static LPALBUFFERCALLBACKSOFT alBufferCallbackSOFT;
 static ALuint callback_buffer;
 static ALuint callback_source;
 
-#define SQUARE_WAVE_AMP 0x2000
+#define SQUARE_WAVE_AMP 0x1000 // Chocolate Doom: 0x2000
```

Comment:
Why this change (`SQUARE_WAVE_AMP` from `0x2000` to `0x1000`)?

Reply:
From the commit comment: "It was way too loud in comparison to the other sound modules." Square waves are loud!